jupyter-nextflow/nextflow-welcome.html

```diff
@@ -39,21 +39,30 @@ <h1 class=header>Welcome to the Nextflow Workspace</h1>
   </ul>
   <h2 class=header>Get started with Nextflow</h2>
   <p>If you are new to Nextflow, visit <a
-      href=https://www.nextflow.io>nextflow.io</a> for detailed information.
+      href=https://www.nextflow.io>nextflow.io</a> for detailed information. 
+    You can also learn more about using Nextflow in BRH Workspace in the BRH Documentation <a
+      href=https://uc-cdis.github.io/BRH-documentation/01-home/</a>. 
```

Comment:
The link isn't right, it made the whole text a link. Please verify before creating or approving PRs @smvgarcia @nss10 🙏 
![Screenshot 2024-04-29 at 3 11 16 PM](https://github.com/uc-cdis/containers/assets/4224001/b08ba67f-2de7-4353-833f-1952bd6fa14b)

Reply:
Which part of the link is incorrect?

Reply:
Sorry. I don't know html, and wasn't sure how to test it.

Reply:
@smvgarcia you can test it by git-cloning the repo and opening `nextflow-welcome.html` in your browser

Reply:
i believe it's missing a character. `<a` instead of `<a>`.

Reply:
OK, looks like I fixed it. I didn't have the text that was to be hyperlinked in there

Reply:
Oh! My bad. I see it now.